packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -143,6 +145,20 @@ export default function DataviewsTemplates() {
 			per_page: -1,
 		} );
 
+	const allAuthorElements = useMemo( () => {
```

Comment:
This could be named just `authors`, I guess?

Reply:
Suggestion applied 👍